proposals/simd/ImplementationStatus.md

```diff
@@ -145,23 +145,23 @@
-| `i16x8.narrow_i32x4_s`     |                           | :heavy_check_mark: |                    |                    |
-| `i16x8.narrow_i32x4_u`     |                           | :heavy_check_mark: |                    |                    |
-| `i16x8.widen_low_i8x16_s`  |                           | :heavy_check_mark: |                    |                    |
-| `i16x8.widen_high_i8x16_s` |                           | :heavy_check_mark: |                    |                    |
-| `i16x8.widen_low_i8x16_u`  |                           | :heavy_check_mark: |                    |                    |
-| `i16x8.widen_high_i8x16_u` |                           | :heavy_check_mark: |                    |                    |
-| `i32x4.widen_low_i16x8_s`  |                           | :heavy_check_mark: |                    |                    |
-| `i32x4.widen_high_i16x8_s` |                           | :heavy_check_mark: |                    |                    |
-| `i32x4.widen_low_i16x8_u`  |                           | :heavy_check_mark: |                    |                    |
-| `i32x4.widen_high_i16x8_u` |                           | :heavy_check_mark: |                    |                    |
+| `i8x16.narrow_i16x8_s`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i8x16.narrow_i16x8_u`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.narrow_i32x4_s`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.narrow_i32x4_u`     |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.widen_low_i8x16_s`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.widen_high_i8x16_s` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.widen_low_i8x16_u`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i16x8.widen_high_i8x16_u` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i32x4.widen_low_i16x8_s`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i32x4.widen_high_i16x8_s` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i32x4.widen_low_i16x8_u`  |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| `i32x4.widen_high_i16x8_u` |                           | :heavy_check_mark: | :heavy_check_mark: |                    |
 
 [1] Tip of tree LLVM as of April 24, 2019
 
 [2] Tested on V8 7.5.0 (candidate). Requires flag `--experimental-wasm-simd`
 
-[3] Tip of tree WAVM as of April 30, 2019
+[3] Tip of tree WAVM as of July 10, 2019. Requires flag `--enable prestd-simd`
```

Comment:
That flag flag in the footnote looks like it may be a typo. Otherwise LGTM.

Reply:
It's not a typo: it's meant to convey that the feature is "pre-standardization". Perhaps that's not very clear, though.